include/deal.II/physics/physics.h

```diff
@@ -0,0 +1,137 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2016 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+#ifndef dealii__physics_h
+#define dealii__physics_h
+
+// This file only provides definition and documentation of the
+// namespace Physics. There is no necessity to include it
+// anywhere in a C++ code. Only doxygen will make use of it.
```

Comment:
We have a number of headers like this, and have typically kept them in `doc/doxygen/headers`. Want to move this one there as well? This way you don't need to bother with the `#ifndef` above, etc.